kernel/src/lib.rs

```diff
@@ -13,14 +13,14 @@
 
 #[macro_use]
 pub mod common;
+pub mod component;
 #[macro_use]
 pub mod debug;
-pub mod hil;
-pub mod ipc;
-
 mod callback;
 mod driver;
 mod grant;
+pub mod hil;
+pub mod ipc;
```

Comment:
Why move these two lines? Weren't they set this way to have public and private modules separated?